src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -16,7 +16,7 @@ class GrowableTable extends React.Component {
     this.handleEdit = this.handleEdit.bind(this);
     this.handleSave = this.handleSave.bind(this);
     this.handleRemove = this.handleRemove.bind(this);
-    this.state = {};
+    this.state = { edited: false };
```

Comment:
It's very confusing. I'm not sure we have it all sorted out. I can try to lay out what the intent is.